phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/db/payments/MetadataTypes.kt

```diff
@@ -289,14 +288,6 @@ data class WalletPaymentMetadataRow(
             && user_notes == null
     }
 
-    /**
-     * This function exists because the `freeze()`
-     * function isn't exposed to iOS.
-     */
-    fun copyAndFreeze(): WalletPaymentMetadataRow {
-        return this.freeze()
-    }
```

Comment:
@robbiehanson Can you check that this method was not used anymore - since we are using the new memory model, there's no need to freeze objects anymore.